platform/ext/target/nordic_nrf/nrf5340/spm_hal.c

```diff
@@ -0,0 +1,278 @@
+/*
+ * Copyright (c) 2018-2020, Arm Limited. All rights reserved.
+ * Copyright (c) 2020, Nordic Semiconductor ASA. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ */
+
+#include <stdio.h>
+#include "cmsis.h"
+#include "platform/include/tfm_spm_hal.h"
+#include "spm_api.h"
+#include "spm_db.h"
+#include "tfm_platform_core_api.h"
+#include "target_cfg.h"
+#include "spu.h"
+#include "mpu_armv8m_drv.h"
+#include "region_defs.h"
+#include "secure_utilities.h"
+
+/* Get address of memory regions to configure MPU */
+extern const struct memory_region_limits memory_regions;
+
+struct mpu_armv8m_dev_t dev_mpu_s = { MPU_BASE };
```

Comment:
Why is it not `static`?
And is it supposed in some future to be used by more functions than `tfm_spm_mpu_init`? If not, it could me moved there.

Reply:
i think it can be made static.
Yes, it should be used for the function that I have not yet implemented :), see TODO, FIXME

Reply:
I made this static now.